src/collectors/windows.plugin/metadata.yaml

```
@@ -0,0 +1,265 @@
+plugin_name: windows.plugin
+modules:
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibProcesses
+      monitored_instance:
+        name: System statistics
+        link: "https://www.microsoft.com/en-us/windows"
+        categories:
+          - data-collection.windows-systems
+        icon_filename: "windows.svg"
+      related_resources:
+        integrations:
+          list: []
+      info_provided_to_referring_integrations:
+        description: ""
+      keywords:
+        - process counts
+        - threads
+        - context switch
+      most_popular: false
+    overview:
+      data_collection:
+          metrics_description: |
+            Perflib provides different statistical methods about Microsoft Windows environment. This collector query for
+            Object 'System' to show actual number of processes, threads and context switches.
+          method_description: ""
```

```suggestion
        metrics_description: |
          This collector monitors the current number of processes, threads, and context switches on Windows systems.
        method_description: |
          It queries the 'System' object from Perflib in order to gather the metrics.
```

Don't leave empty overview fields, also I think the file is not formatted, so I pushed back the indentation